README.md

```diff
@@ -93,7 +93,7 @@ New major version indicates that there is an incompatible breaking change.
 > [!WARNING]
 > Please read carefully release notes of this chart before upgrading.
 
-### A standard installation
+### Upgrade traefik chart
```

Comment:
Maybe explicitly call this `Upgrade traefik chart with manual CRD management` or `without opt-in CRDs charts` for clarity

Reply:
I tried something with cf41d2f20fb7c182a45c825d37645bd3788830d4. Wdyt ?

Reply:
Yeah, that looks great to me!